build.fsx

```diff
@@ -67,6 +67,7 @@ Target "Test_Solution" (fun _ -> test "PostalRegistry")
 
 Target "Publish_Solution" (fun _ ->
   [
+    "PostalRegistry.Api.Projector"
```

Comment:
Can you rename `PostalRegistry.Api.Projector` to `PostalRegistry.Projector` please? (also in the solution and on disk)